### DISCUSSION TITLE
Bridge version from mmm-refactor to meeting-media-manager

### DISCUSSION THREAD

**Discussion by mtdvlpr** [sircharlo/meeting-media-manager#3654] [Ideas] [answered]
### Is your idea related to a problem?

It seems people are still using the mmm-refactor version. Maybe a 24.10.6 release on the refactor repo with a deprecation banner will be beneficial.

See https://jw-projects.sentry.io/issues/5773493798/?project=4507449197920256

### Describe your idea

To release one more version in the mmm-refactor repo with a persistent banner to switch to the official release. Similar to what was done to the legacy version before the refactor was released.

### Describe alternatives you've considered

n.a.

### Confirmations

- [X] I confirm that I have read the documentation website.
- [X] I confirm that I have searched through the existing issues to make sure my idea is not a duplicate.
- [X] I confirm that I have searched through the existing discussions to make sure my idea is not a duplicate.

**Comment by mtdvlpr** [comment]
@sircharlo, thoughts?

**Reply by sircharlo** [reply, marked as answer]
I agree. I hope to have time to do this this week.
